tests/src/Provider/FacebookTest.php

```diff
@@ -28,7 +30,7 @@ class FacebookTest extends TestCase
      */
     protected $provider;
 
-    protected function setUp()
+    protected function setUp(): void
```

Comment:
For instance, this is not compatible with PHP 5.6. I know that it won't impact runtime, but tests won't run on 5.6.